ajax/libs/angular-scroll/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "angular-scroll",
+  "description": "Scrollspy, animated scrollTo and scroll events",
+  "author": "Joel Arvidsson",
+  "homepage": "https://github.com/oblador/angular-scroll",
+  "keywords": [
+    "angular",
+    "smooth-scroll",
+    "scrollspy",
+    "scrollTo",
+    "scrolling"
+  ],
+  "filename": "angular-scroll.min.js",
+  "version": "0.6.4",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oblador/angular-scroll.git"
+  },
+  "bugs": {
+    "url": "https://github.com/oblador/angular-scroll/issues"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://git@github.com:oblador/angular-scroll.git",
```

Comment:
OK, This is getting tiresome.  What is in the wrong format.  I tried this before and was told that the URL was in the wrong format and I changed it.  You need to get more descriptive on what's wrong.

Look at this https://github.com/cdnjs/cdnjs/pull/4171 where he is telling me it is the wrong format.